odoo/addons/base/models/res_users.py

```diff
@@ -697,6 +701,57 @@ def _has_group(self, group_ext_id):
     # for a few places explicitly clearing the has_group cache
     has_group.clear_cache = _has_group.clear_cache
 
+    def _get_access_count(self):
+        for user in self:
+            groups = user.groups_id
+            user.access_count = len(groups.mapped('model_access'))
+            user.rules_count = len(groups.mapped('rule_groups'))
+            user.groups_count = len(groups)
+
+    def action_show_groups(self):
+        self.ensure_one()
+        self.user_has_groups('base.group_erp_manager')
```

Comment:
user_has_groups returns a boolean. This line has therefore no effect. I suppose you want to raise or return something ? Is it necessary as it should be covered by access rights ?